### ISSUE TITLE
ERROR: Unable to download JSON metadata: HTTP Error 422: Unprocessable Entity (caused by HTTPError())

### ISSUE THREAD

**Issue by NomaanMalick** [ytdl-org/youtube-dl#29459]
## Checklist

- [ x] I'm reporting a broken site support
- [ x] I've verified that I'm running youtube-dl version **2021.06.06**
- [x ] I've checked that all provided URLs are alive and playable in a browser
- [x ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.viki.com/videos/1080865v', '--cookies', 'C:\\Users\\Nomaan\\Desktop\\cookies.txt']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2021.06.06
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.19041
[debug] exe versions: ffmpeg 4.4-full_build-www.gyan.dev, ffprobe 4.4-full_build-www.gyan.dev
[debug] Proxy map: {}
[viki] 1080865v: Downloading video JSON
ERROR: Unable to download JSON metadata: HTTP Error 422: Unprocessable Entity (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpkqxnwl31\build\youtube_dl\extractor\common.py", line 634, in _request_webpage
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpkqxnwl31\build\youtube_dl\YoutubeDL.py", line 2288, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default
```

## Description

Is there any way to fix this? Or does youtube-dl not support Viki anymore?

**Comment by NicolasFlandrois** [comment]
I got the same error messages, same website.
Any clues or input would be a life changer.

**Comment by zackmark29** [comment]
> I got the same error messages, same website.
> Any clues or input would be a life changer.

Just wait a little.
https://github.com/yt-dlp/yt-dlp/pull/473

**Comment by zackmark29** [comment]
binary released
https://github.com/zackmark29/yt-dlp/releases/tag/2021.07.08

**Comment by rmiddle** [comment]
I am seeing the same issue.  Do I need to post a dump file to help troubleshoot the issue?

**Comment by rmiddle** [comment]
This is a duplicate of https://github.com/ytdl-org/youtube-dl/issues/29502 and see the latest comment for a fix.

Thanks
Robert

**Comment by issues101** [comment]
> This is a duplicate of #29502 and see the latest comment for a fix.
Also see a later comment why it is not a fix.

**Comment by issues101** [comment]
> This is a duplicate of #29502 and see the latest comment for a fix.
Also see a later comment on why this is not a fix.

**Comment by dirkf** [comment]
Fixed in #30596.
